RecoLocalCalo/EcalRecAlgos/interface/Common.h

```diff
@@ -0,0 +1,38 @@
+#ifndef RecoLocalCalo_EcalRecAlgos_interface_Common_h
+#define RecoLocalCalo_EcalRecAlgos_interface_Common_h
+
+#include <cstdint>
+#include <cmath>
+#include <cassert>
+
+#include <cuda.h>
+#include <cuda_runtime.h>
+
+// a workaround for std::abs not being a constexpr function
+namespace ecal {
+
+template<typename T>
```

Comment:
I agree that it would be useful to have this.

Can you add it in `HeterogeneousCore/CUDAUtilities/interface/cudastdAlgorithm.h` instead of here ?

Reply:
I would prefer to do the following:
- get #335 merged
- short pr to refactor that just quickly...